sdk/communication/Azure.Communication.Calling.Server/src/ConversationClient.cs

```diff
@@ -0,0 +1,622 @@
+                    conversationId,
+                    participantsInternal,
+                    alternateCallerIdInternal,
+                    operationContext,
+                    callbackUri?.AbsoluteUri,
+                    cancellationToken
+                    );
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Add participant to the call.
+        /// </summary>
+        /// <param name="conversationId"> The conversation id that can be a group id or a encoded conversation url retrieve from client. </param>
+        /// <param name="participant"> The identity of participant to be added to the call. </param>
+        /// <param name="callbackUri"></param>
+        /// <param name="alternateCallerId">The phone number to use when adding a pstn participant.</param>
+        /// <param name="operationContext">The operation context.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        public virtual async Task<Response> AddParticipantAsync(string conversationId, CommunicationIdentifier participant, Uri callbackUri, string alternateCallerId = default, string operationContext = default, CancellationToken cancellationToken = default)
+        {
+            using DiagnosticScope scope = _clientDiagnostics.CreateScope($"{nameof(ConversationClient)}.{nameof(AddParticipantAsync)}");
+            scope.Start();
+            try
+            {
+                Argument.AssertNotNull(participant, nameof(participant));
+
+                var participantsInternal = new List<CommunicationIdentifierModel> { CommunicationIdentifierSerializer.Serialize(participant) };
+                var alternateCallerIdInternal = string.IsNullOrEmpty(alternateCallerId) ? null : new PhoneNumberIdentifierModel(alternateCallerId);
+                return await RestClient.InviteParticipantsAsync(
+                    conversationId,
+                    participantsInternal,
+                    alternateCallerIdInternal,
+                    operationContext,
+                    callbackUri?.AbsoluteUri,
+                    cancellationToken
+                    ).ConfigureAwait(false);
+            }
+            catch (Exception ex)
+            {
+                scope.Failed(ex);
+                throw;
+            }
+        }
+
+        /// <summary>
+        /// Remove participant from the call.
+        /// </summary>
+        /// <param name="conversationId"> The conversation id that can be a group id or a encoded conversation url retrieve from client. </param>
+        /// <param name="participantId">The participant id.</param>
+        /// <param name="cancellationToken">The cancellation token.</param>
+        public virtual Response RemoveParticipant(string conversationId, string participantId, CancellationToken cancellationToken = default)
```

Comment:
Same comment here, we take in type `CommunicationIdentifier` when we add participant but take in `string` when removing. I think these should be consistent

Reply:
participant id is added because a person can login with same identity from multiple devices, each login have different participant id but with same identity.

 We will add support in near future, if we use identity, all device login kicked out